tests/python/pants_test/engine/test_rules.py

```diff
@@ -201,6 +202,32 @@ def goal_rule_returning_a_non_goal() -> int:
         return 0
 
 
+def fmt_task_func(func: Callable) -> str:
+  """Generate the str for a Rust Func, which is how Rust refers to `@rule`s."""
```

Comment:
imho this could be replaced with the path to the appropriate file in the rust codebase that defines and uses `Func`, or perhaps a link to github of some region highlighting the relevant part? This feels maybe vague and possibly brittle (the name "Func" might change).

Reply:
Agreed. I felt weird calling this a `task_func` but didn't know what else to call it. We need it to be different than `fmt_rule` for the sake of `test_engine.py` and `test_scheduler.py`.

Maybe a permalink to the Rust code to reduce the risk of that Rust code moving and now having a stale comment?

Reply:
yes!!!!